composer.json

```diff
@@ -20,7 +20,8 @@
         "vincentlanglet/twig-cs-fixer": "^0.4.1 || ^0.5",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
         "phpstan/extension-installer": "^1.1",
-        "phpcompatibility/php-compatibility": "^9.3"
+        "phpcompatibility/php-compatibility": "^9.3",
+        "guzzlehttp/guzzle": "^6.0"
```

Comment:
I wanted to make this more generic, but wasn't sure how. Ideas are welcome.

Reply:
Will this work with different versions of Guzzle? I ask because of concern over whether there might be other dependencies that require different versions. The wider our requirement can be, the better.

Reply:
Dropping would be good. So would allowing 7.x if our code would work with that!